.github/workflows/docs-pull-request.yml

```diff
@@ -0,0 +1,38 @@
+name: Docs Lint/Link Checks
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This workflow yaml file is called `docs-pull-request.yml` but we're running it on pushes to the main branch? Something needs to change here.

Reply:
Updated